clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -264,3 +264,16 @@
   CmakeSyntax: add_executable_opts
   In: add_executable(${value} OPTIONS ${options})
   Out: add_executable(${value})
+
+- Rule: rule_cuda_compile_fatbin
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: cuda_compile_fatbin
+  In: cuda_compile_fatbin(${value})
```

Comment:
According the definition of cuda_compile_fatbin:
```
cuda_compile_fatbin(<generated_files> <file>... [OPTIONS ...])
```
The "[OPTIONS ...]" should also be considered, pls add a new yaml rule to cover this scenarios, like Line 264 does.

Reply:
Done.